tests/fixes/mod.rs

```diff
@@ -54,30 +54,96 @@ fn key_without_value() {
 }
 
 #[test]
-fn unfixed_warnings() {
+fn unordered_key() {
```

Comment:
Please move this test to the `unordered_key.rs` file.

Reply:
Done